src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java

```diff
@@ -0,0 +1,395 @@
+            Arrays.asList(BigDecimal.valueOf(100), BigDecimal.valueOf(90), BigDecimal.valueOf(60),
+                          BigDecimal.valueOf(50), BigDecimal.valueOf(40), BigDecimal.valueOf(25));
+    private static final List<DuplicationMapping> DUPLICATION_LEVELS =
+            Arrays.asList(new DuplicationMapping(BigDecimal.valueOf(3), "3"),
+                          new DuplicationMapping(BigDecimal.valueOf(5), "5"),
+                          new DuplicationMapping(BigDecimal.TEN, "10"),
+                          new DuplicationMapping(BigDecimal.valueOf(20), "20"));
+
+    private final BranchDetails branchDetails;
+    private final MeasuresHolder measuresHolder;
+    private final PostAnalysisIssueVisitor postAnalysisIssueVisitor;
+    private final QualityGate qualityGate;
+    private final Analysis analysis;
+    private final Project project;
+    private final Configuration configuration;
+
+    AnalysisDetails(BranchDetails branchDetails, PostAnalysisIssueVisitor postAnalysisIssueVisitor,
+                    QualityGate qualityGate, MeasuresHolder measuresHolder, Analysis analysis, Project project,
+                    Configuration configuration) {
+        super();
+        this.branchDetails = branchDetails;
+        this.measuresHolder = measuresHolder;
+        this.postAnalysisIssueVisitor = postAnalysisIssueVisitor;
+        this.qualityGate = qualityGate;
+        this.analysis = analysis;
+        this.project = project;
+        this.configuration = configuration;
+    }
+
+    public String getBranchName() {
+        return branchDetails.getBranchName();
+    }
+
+    public String getCommitSha() {
+        return branchDetails.getCommitId();
+    }
+
+    public QualityGate.Status getQualityGateStatus() {
+        return qualityGate.getStatus();
+    }
+
+    public String createAnalysisSummary(FormatterFactory formatterFactory) {
+
+        BigDecimal newCoverage =
+                findQualityGateCondition(CoreMetrics.NEW_COVERAGE_KEY).map(QualityGate.Condition::getValue)
+                        .map(BigDecimal::new).orElse(null);
+
+        double coverage = findMeasure(CoreMetrics.COVERAGE_KEY).map(MeasureWrapper::getDoubleValue).orElse(0D);
+
+        BigDecimal newDuplications = findQualityGateCondition(CoreMetrics.NEW_DUPLICATED_LINES_DENSITY_KEY)
+                .map(QualityGate.Condition::getValue).map(BigDecimal::new).orElse(null);
+
+        double duplications =
+                findMeasure(CoreMetrics.DUPLICATED_LINES_DENSITY_KEY).map(MeasureWrapper::getDoubleValue).orElse(0D);
+
+        NumberFormat decimalFormat = new DecimalFormat("#0.00");
+
+        Map<RuleType, Long> issueCounts = countRuleByType();
+        long issueTotal = issueCounts.values().stream().mapToLong(l -> l).sum();
+
+        List<QualityGate.Condition> failedConditions = findFailedConditions();
+
+        String baseImageUrl = configuration.get("sonar.pullrequest.base.image.url")
```

Comment:
Any reason you've changed this from `com.github.mc1arke.sonarqube.plugin.branch.image-url-base`?

Reply:
just to for consistency in the PropertyDefinition but as you outlined your preferred naming convention I will change this again.